homework-3/scripts/renderTriangle.js

```diff
@@ -0,0 +1,45 @@
+function createTriangle(num) {
```

Comment:
better this function be pure function, without side effects, 
meaning its call shouldn't change anything. 
So let it return result, without knowing about anything about dom.